lib/perl/Genome/Db/Ensembl/Command/Run/Base.pm

```diff
@@ -669,4 +692,16 @@ sub cache {
     return $cache_result;
 }
 
+sub _result_users {
+    my $self = shift;
+
+    my $analysis_build_anp = $self->analysis_build?
+        $self->analysis_build->model->analysis_projects : undef;
+
+    return {
+        requestor => ($self->analysis_build // $self->analysis_process),
+        sponsor => ($self->analysis_project // $analysis_build_anp // Genome::Sys->current_user),
```

Comment:
I make no claims as to the value; only that we accept it if it is defined.  I believe that `//` is more correct in these situations, as we're not performing a logical test of the values.  (You're right that a false-valued build or analysis project isn't something that is likely to exist.)